node_client_example/nginx.conf

```diff
@@ -23,8 +23,11 @@ http {
     listen                          80;
     server_name                     kernel_gateway;
 
-    location / {
-      proxy_pass http://kernel_gateway;
+    location /jupyter/v1/12345678-1234-1234-1234-123456789012/ {
```

Comment:
Just to demonstrate arbitrary proxy paths work fine (e.g., in a multi-tenant environment).